docs/Editing/Auto-Suggest.md

```diff
@@ -182,11 +182,17 @@ Our second task looks like this. Note that the due date is shown in the Descript
 
 ![auto-suggest-edit-incorrect-task](../images/auto-suggest-edit-incorrect-task.png)
 
-### How do I see more suggestions?
+### How do I see less/more suggestions?
```

Comment:
Just noting some grammatical things I plan to try and fix...

[it's 'fewer or more' not ](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://www.merriam-webster.com/grammar/fewer-vs-less%23:~:text%3DIt%2520goes%2520like%2520this%253A%2520fewer,%2522%2520and%2520%2522less%2520effort.%2522&ved=2ahUKEwjw7IOLveiGAxV0V0EAHas_CRIQFnoECBMQAw&usg=AOvVaw0aMnnz6J3QM2ADak5Lhr5z)

> Generally, fewer is used when a number of things can be counted ("fewer problems") and less is used when a number is measured ("less trouble" or "less time"). However, this is not a strict rule and there are accepted instances of less being used with countable amounts such as "250 words or less," "3 items or less," and especially with money ("less than $20") and distance ("less than 3 miles"). Note that in all these cases, the things considered are often thought of as amounts rather than numbers.

So I am inclined to change it to 'fewer or more'

Reply:
> So I am inclined to change it to 'fewer or more'

Done.